optimade/models/entries.py

```diff
@@ -73,13 +73,13 @@ class EntryResourceAttributes(Attributes):
 - **Type**: timestamp.
 - **Requirements/Conventions**:
 
-  - **Response**: REQUIRED in the response unless explicitly excluded.
-  - **Query**: MUST be a queryable property with support for all mandatory filter operators.
+  - **Support**: SHOULD be supported by all implementations, i.e., SHOULD NOT be :val:`null`.
```

Comment:
I guess this is taken from the spec?
From a client developer perspective, what's the difference between OPTIONAL and SHOULD? 
Since you can't rely on either, is this distinction necessary?

Reply:
I guess from a client POV SHOULD is equal to MUST in expectation level? Although one should perhaps have a fallback? I don't really know. But since this is not a client, but a server, we definitely SHOULD have this field :)